tests/system/test_integration_acm.py

```diff
@@ -254,3 +274,91 @@ def test_log_kill_switch_active(self):
             "error": "forbidden request: endpoint is disabled",
             "response_code": 403,
         }, config_request_logs[0])
+
+
+class RumAgentConfigurationIntegrationTest(AgentConfigurationTest):
+    config_overrides = {
+        "enable_rum": "true",
+    }
+
+    def test_rum(self):
+        service_name = "rum-service"
+        create_config_rsp = self.create_service_config({"transaction_sample_rate": 0.2}, service_name, agent="rum-js")
```

Comment:
following up on your abbreviation preferences, don't use `rsp` as abbrev for `response` (I'd be totally fine with `resp` as that's widely used).

Reply:
yes, but this is copied over and the same abbreviation is all over the place. i didn't suggest to remove all abbreviations from the code base in one go.